package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "city-in-the-south",
+  "version": "1.0.0",
+  "description": "## PROBLEM:",
+  "main": "index.js",
+  "scripts": {
+    "server": "fuser -k 5000/tcp;clear; nodemon server/index.js",
```

Comment:
```suggestion
    "start:server": "fuser -k 5000/tcp;clear; nodemon server/index.js",
```
I think this more cleanliness and understanding

Reply:
I think it is ok, no need for extra word!

Reply:
true, no need for extra word
though I am not sure about killing the port before starting the server, it gives an error.
![image](https://user-images.githubusercontent.com/18149438/58366490-a0124400-7edb-11e9-9905-de4452beaa66.png)

Reply:
I just used this technique to insure the port is not used, try to push ctrl +z to exit without killing the port then start again, you can see the difference.

Reply:
@AymanAlqoqa I need to understand the script works, so I think we need an extra word ...

Reply:
@Fatmasiam  it's not a big deal, and I agree with Ayman that the fewer words the faster to type the script
Also, `npm run server` means that I want to run the server
`npm run client` also means I want to run the client
`npm run dev` I want to run the development mood

looks pretty clear to me :thinking: